1-documentation/profiles/usage.md

```diff
@@ -0,0 +1,18 @@
+## Why do we need profiles?
+To change configuration in `.properties` files easily for different environments.
+
+## What profiles are present in application?
+At the moment there are two profiles if our application:
+1. `cloud` - for production environment with appropriate configuration in `application-cloud.properties`. 
+2. `debug` - for test environment with appropriate configuration in `application-debug.properties`.
+
+These profiles are situated in each service.
+
+## How can we run application with specific profile?
+Append parameter -P`profile-name` while packaging an application.
+For example:
+
+```mvn clean package -Pcloud```
```

Comment:
this will change profile for building only, but what about running application in IntellijIdea/ Eclipse? 

test that everything is worked, e.g. `server.port=8000` for debug and `server.port=9000` for cloud. 

Launch Starter and check if server port changes as profile is chaged

Reply:
Replaced maven configuration with JVM System Parameter.